client/ayon_core/modules/deadline/plugins/publish/submit_houdini_render_deadline.py

```diff
@@ -289,7 +292,7 @@ def get_plugin_info(self, job_type=None):
         instance = self._instance
         context = instance.context
 
-        hou_major_minor = hou.applicationVersionString().rsplit(".", 1)[0]
+        self.hou_major_minor = hou.applicationVersionString().rsplit(".", 1)[0]
```

Comment:
I really don't like us putting this on `self` here. Can we just pass `version` to the `_get_husk_standalone_plugin_info` function explicitly?

Reply:
Implemented this upstream currently. Thanks!